rules/xcodeproj.bzl

```diff
@@ -310,10 +339,22 @@ def _gather_asset_sources(target_info, path_prefix):
                 "buildPhase": "none",
             }
             asset_sources.append(payload)
-    for datamodel_short_path in datamodel_groups.keys():
+
+    for datamodel_key in datamodel_groups.keys():
+        datamodel_path = datamodel_groups[datamodel_key]
+        payload = {
+            "path": paths.join(path_prefix, datamodel_path),
+            "group": paths.dirname(datamodel_path),
+            "optional": True,
+            "buildPhase": "none",
+        }
+        asset_sources.append(payload)
+
+    for asset_key in catalog_groups.keys():
```

Comment:
same as above, iterate over the `items()` tuples

Reply:
keeping as is since the values are not that important now, now use dict just to de-duplciate the keys)